addons/zeus/ACE_Settings.hpp

```diff
@@ -0,0 +1,23 @@
+class ACE_Settings {
+    class GVAR(zeusAscension) {
+        typeName = "BOOL";
+        value = 1;
```

Comment:
In my opinion, all this settings should be off by default. (No cues for ascension, no bird, no radioOrdnance and no reveal mines).